server/routes/__testutils/appSetup.ts

```diff
@@ -7,24 +7,17 @@ import errorHandler from '../../errorHandler'
 import * as auth from '../../authentication/auth'
 import type { Services } from '../../services'
 import AuthRole from '../../enumeration/authRole'
+import { User } from '../../@types/CvlUserDetails'
```

Comment:
> not sure this is right?
> 
> Shouldn't this be of type `LocalsUser` which is a combination of `User` and `CvlUserDetails`?
> 
> Why did you need to change this?

Yes, this User is pointing to LocalsUser only.
export type User = Express.LocalsUser;
Instead of Using LocalUser in all places we decided to assign it to the User as many places in the application is pointing to this, if you remember.